test/test_unary_ufuncs.py

```diff
@@ -75,6 +75,15 @@ def generate_tensors_from_vals(vals, device, dtype, domain):
     assert len(vals) < _medium_length  # medium tensor should contain all vals
     assert _medium_length % 4 == 0  # ensure vectorized code coverage
 
+    if not dtype.is_complex:
+        # Filter values based on Operators domain.
+        # Note: Complex numbers don't belong to ordered field,
+        #       so we don't filter for them.
+        if domain[0] is not None:
+            vals = list(filter(lambda x: x > domain[0], vals))
```

Comment:
domain is [low, high) (or it should be):

https://github.com/pytorch/pytorch/blob/1145e2c6e238058d0f4a87e90db95c22a9dc2a87/torch/testing/_internal/common_methods_invocations.py#L366

so the first check should be `x >= domain[0]`?